README.md

```diff
@@ -2,18 +2,12 @@
 
 HttpEx is an Elixir HTTP abstraction library to easily log, trace and mock your HTTP calls.
 
-Note: work in progress. Things to come:
-
-* Add support for different HTTP clients (HTTPoison, Finch)
-* Add support for different tracing backends (currently only supports OpenTelemetry)
-* Add support for different logger backends
```

Comment:
I don't think these are specific goals for the lib. If this comes up as a necessity we can look at it.